GAD-7.json

```diff
@@ -26,6 +38,14 @@
         {
           "code": "69725-0",
           "system": "http://loinc.org"
+        },
+        {
+          "code": "398",
+          "system": "http://cnics-pro.cirg.washington.edu"
+        },
+        {
+          "code": "1254",
+          "system": "http://paintracker.cirg.washington.edu"
```

Comment:
@ivan-c  @pbugni  I'm likely to abandon these item-level mappings - doesn't seem worth the effort, the use case for consuming these isn't well defined.

Reply:
It does muddy the waters.  We do need an ETL consumable map somewhere, but given the desire to use the `Questionnaire`s themselves across multiple products, such a client specific map may be better elsewhere.